vendor/github.com/containers/common/pkg/config/default.go

```diff
@@ -105,8 +105,6 @@ const (
 	DefaultApparmorProfile = apparmor.Profile
 	// SystemdCgroupsManager represents systemd native cgroup manager
 	SystemdCgroupsManager = "systemd"
-	// DefaultLogDriver is the default type of log files
-	DefaultLogDriver = "k8s-file"
```

Comment:
Why was this removed?  The cursor bug was happening for a while, but was fixed by some of the logdriver changes, which seem to be being partially reveted.